build.properties

```diff
@@ -3,8 +3,9 @@
 #
 # Minecraft / Forge.
 minecraft_version=1.12.2
-forge_version=14.23.5.2836
+forge_version=14.23.5.2768
```

Comment:
any particular reason to rollback the forge version?

Reply:
My setup dint want to build so I went to stable and it worked can try to change it back to the version you used now

Reply:
Yeah, we used a higher version on purpose because there is a forge fix in there we need.